### PR TITLE
reformat members.md so it's easier to check

### PR DESCRIPTION
**Pull request by colinbrislawn**
part of #54

### CODE REVIEW

File: website/members.md [hunk 1]
```diff
@@ -32,42 +33,31 @@ not limited to:
...
-Dalhousie University, Canada. Quince, Christopher, cq8u@udcf.gla.ac.uk, 
-University of Glasgow, UK Scyzrba, Alex, asczyrba@lbl.gov, DOE Joint
-Genome Institute Shade, Ashley, (J. Handlesman group), Yale Sullivan,
-Matthew, [mbsulli@email.arizona.edu,](mailto:mbsulli@email.arizona.edu)
-University of Arizona, USA Sun, Fengzhu,
-[fsun@college.usc.edu,](mailto:fsun@college.usc.edu)University of
-Southern California, USA. Teske, Andreas, University of North Carolina,
-Chapel Hill
+- Balaji, Pavan, <balaji@mcs.anl.gov>, Argonne National Laboratory and
+University of Chicago, USA.
```
Comment (josenavas) [inline website/members.md]: Can you add this to the above line - so it's a single line like all the others?

**Comment by josenavas** [comment]
@ackermag your input here would be really useful too.

**Comment by josenavas** [comment]
I would prefer to wait until @ackermag reviews this list. @cuttlefishh the branch is already created (this PR is against that branch in upstream)

**Comment by ackermag** [comment]
I am not sure how to complete this task - @colinbrislawn can you make text
available to me or point me to it (sorry I am a naive user)


On Mon, Mar 14, 2016 at 10:02 PM, Jose Navas <notifications@github.com>
wrote:

> I would prefer to wait until @ackermag <https://github.com/ackermag>
> reviews this list. @cuttlefishh <https://github.com/cuttlefishh> the
> branch is already created (this PR is against that branch in upstream)
>
> —
> You are receiving this because you were mentioned. Reply to this email
> directly or view it on GitHub: <a href="
> https://github.com/biocore/emp/pull/56#issuecomment-196662325">
> https://github.com/biocore/emp/pull/56#issuecomment-196662325</a>
>



-- 
Gail Ackermann
Knight Lab
UCSD
glackermann@ucsd.edu <ackermag@ucsd.edu>

**Comment by josenavas** [comment]
@ackermag just click in the "Files changed" tab and look at the text there, if you know that there is somebody missing or there is somebody that shouldn't be there, please comment in line. I can't show you tomorrow if you need to.

**Comment by josenavas** [comment]
I meant I CAN show you tomorrow - starting to get late....

**Comment by ackermag** [comment]
That is okay - i may have to run by Jack and Rob so not going to happen
soon.

On Mon, Mar 14, 2016 at 10:33 PM, Jose Navas <notifications@github.com>
wrote:

> I meant I CAN show you tomorrow - starting to get late....
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emp/pull/56#issuecomment-196671809
>



-- 
Gail Ackermann
Knight Lab
UCSD
glackermann@ucsd.edu <ackermag@ucsd.edu>

**Comment by josenavas** [comment]
Thanks @ackermag that's ok
However, we would like to have this merged soon - can you take a quick a look and see if something pops up to you? We can then open issues once we have Rob's and Jack's feedback.

**Pull request merged**